pallets/dbc-price-ocw/src/lib.rs

```diff
@@ -281,4 +281,13 @@ impl<T: Config> DbcPrice for Pallet<T> {
             .checked_mul(&one_dbc)?
             .checked_div(&dbc_price)
     }
+
+    fn get_dlc_amount_by_value(value: u64) -> Option<Self::Balance> {
+        let one_dlc: Self::Balance = 1_000_000_000_000_000_u64.saturated_into();
+        let dlc_price = 3000u64.saturated_into();
```

Comment:
define const for `1_000_000_000_000_000_u64`, and replace it in `get_dbc_amount_by_value`

why hardcode the `dlc_price`?

Reply:
The price of DLC is temporarily 0.003. When DLC has a price, it will be replaced by the real price